src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3183,6 +3183,15 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/DatasetSummaryModel'
+        roleMap:
```

Comment:
is this the interface that you're planning on having services implement?

Reply:
No, I think the first step is to fully implement an API for TDR (and possibly workspaces/rawls too), and then think about what the general version of that API would be.